mongo-compose.yml

```diff
@@ -0,0 +1,7 @@
+# docker compose -f mongo-compose.yml up
+version: "3.9"
+services:
+  mongodb:
+    image: mongo
+    ports:
+      - "27017:27017"
```

Comment:
Is this for people who only want to run mongo on Docker? We can add this to the docs (I can help and do in another PR)

Reply:
It is a way to run mongo so you can run playwright.
It is more of a stopgap measure.

There should be a way to direct playwright to use the existing server from the docker-compose, but I haven't had time to scour the docs for the correct cli option.

Reply:
Sure, sounds good 👍  Thanks @dan-mba

Reply:
I checked the playwright docs.
You can specify a config file with the `--config` option.

I tried this, but it doesn't work because the global-teardown  tries to cleanup the test users in mongo.

Reply:
No worries, these improvements can come in a later PR, what there is now is great 👍 

I will just wait for Emma to review then we can merge